src/shared/components/event-management/services/event-state-machine.service.ts

```diff
@@ -164,6 +173,15 @@ export class EventStateMachineService {
   public entryActionForStateOneTaskAssignedToUser(state: State, context: StateMachineContext): void {
     // Trigger final state to complete processing of state machine
     state.trigger(StateMachineStates.FINAL);
+
+    // Get task assigned to user
+    let task = context.tasks.find(x => x.id === context.taskId);
+    if (task === undefined) {
```

Comment:
```suggestion
    if (!task) {
```

Reply:
Thank you @udaydenduluri33, I have now done the changes as you suggested.